Wireframe/index.html

```diff
@@ -10,23 +10,41 @@
     <header>
       <h1>Wireframe</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        On this page, we are going to explain the three following concepts:
+        1.What is the purpose of a README file?
+        2.What is the purpose of a wireframe?
+        3.What is a branch in Git?
       </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
-        <p>
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam,
-          voluptates. Quisquam, voluptates.
+        <img src="slide6.png" alt="" />
```

Comment:
Can you review the `<img>` element and the two required attributes it must always include? There is something you are missing in all of your `<img>` elements.
https://www.w3schools.com/tags/tag_img.asp